flink-connectors/flink-connector-kafka-0.10/src/main/java/org/apache/flink/streaming/connectors/kafka/Kafka010TableSource.java

```diff
@@ -62,7 +64,8 @@ public Kafka010TableSource(
 			Properties properties,
 			DeserializationSchema<Row> deserializationSchema,
 			StartupMode startupMode,
-			Map<KafkaTopicPartition, Long> specificStartupOffsets) {
+			Map<KafkaTopicPartition, Long> specificStartupOffsets,
+			long startupTimestampMillis) {
```

Comment:
can reuse the param of properties , not add new param startupTimestampMillis?

Reply:
I think we'd better not mix the startup timestamp with the Kafka connector properties.